<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDirectoriesDialog</class>
 <widget class="QDialog" name="EditDirectoriesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit directories dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListView" name="lvDirectories">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Files</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QCheckBox" name="cbFileRead">
                <property name="text">
                 <string>Read</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbFileWrite">
                <property name="text">
                 <string>Write</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbFileDelete">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbFileAppend">
                <property name="text">
                 <string>Append</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Directories</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QCheckBox" name="cbDirCreate">
                <property name="text">
                 <string>Create</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbDirDelete">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbDirList">
                <property name="text">
                 <string>List</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbDirSubDirs">
                <property name="text">
                 <string>+ Subdirs</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbIsHome">
                <property name="text">
                 <string>Home dir</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="btnDelete">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="leDirectory"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditDirectoriesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditDirectoriesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
